js/materialbox.js

```diff
@@ -16,19 +16,62 @@
       var outDuration = 200;
       var origin = $(this);
       var placeholder = $('<div></div>').addClass('material-placeholder');
+      placeholder.css({ display:" table", position: "relative", width: origin.css("width"), height: origin.css("height") }); // for the progress bar to be same width of img
```

Comment:
don't use display: table - You can achieve the same goals using display: block here

Reply:
And also I would not do origin.css ("width") and origin.css ("height"), because someone has an adaptive design.

Reply:
But because of this there is a problem that the placeholder when the image is open has dimensions of 0 to 0 and pictures and other pictures become the place of the one that we called. There is a negative effect.